create-plugin-update/README.md

```diff
@@ -22,28 +25,26 @@ name: Create Plugin Update
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 0 1 * *' # run once a month on the 1st day
-
-# To use the default github token with the following elevated permissions make sure to check:
-# **Allow GitHub Actions to create and approve pull requests** in https://github.com/USER_NAME/REPO_NAME/settings/actions.
-# Alternatively create a fine-grained personal access token for your repository with `contents: read and write` and `pull requests: read and write` and pass it to the action.
-
-permissions:
-  contents: write
-  pull-requests: write
+    - cron: "0 0 1 * *" # run once a month on the 1st day
 
 jobs:
   release:
     runs-on: ubuntu-latest
     steps:
       - uses: grafana/plugin-actions/create-plugin-update@create-plugin-update/v1.1.0
+<<<<<<< Updated upstream
+=======
+        with:
+          token: ${{ secrets.GH_PAT_TOKEN }}
+>>>>>>> Stashed changes
```

Comment:
bad merge here

Reply:
Should be fixed in https://github.com/grafana/plugin-actions/pull/180/commits/29322d0450a21a8c6da28d154bce298538134aee